### PR TITLE
Xanathar domains

### PR DESCRIPTION
**Pull request by oganm**

### CODE REVIEW

File: _posts/2014-08-24-bane.markdown [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: post
 title:  "Bane"
 date:   2014-08-24
 sources: [PHB.216]
-tags: [bard, cleric, paladin (vengeance), level1, enchantment]
+tags: [bard, cleric, cleric (grave), paladin (vengeance), level1, enchantment]
```
Comment (Traneptora) [inline _posts/2014-08-24-bane.markdown]: Don't add domain spells that are on the primary list.

Comment (oganm) [reply _posts/2014-08-24-bane.markdown]: Sure will fix. Any reason why you went with that? People (me) might still be interested in filtering for domain spells. Also you might want to add that to the style guide.

Comment (Traneptora) [reply _posts/2014-08-24-bane.markdown]: Well if you just want a list of spells for a particular domain, the PHB (or equivalent) is a better resource. As for why? It's redundant mostly.

Comment (oganm) [reply _posts/2014-08-24-bane.markdown]: It has few use cases in real life but in general, it is faster to type `cleric (domain)` than fiddling with a book if you are curious which domains get which spells for... reasons. 

I have my own selfish reasons that I use the data in this repo for [automating my own games](https://github.com/oganm/wizaRd), so would prefer having more complete metadata, but I understand that that's not the intention here.

So I'll go back and fix stuff probably tonight, but if you ever change your mind about having the domain and class together, I can add them en masse at a spare time

**Comment by oganm** [comment]
just a reminder, I believe this is done now

**Comment by Traneptora** [comment]
Thanks github, for not notifying me.

**Pull request merged**